data_structures/arrays/find_triplets_with_0_sum.py

```diff
@@ -1,7 +1,7 @@
 from itertools import combinations
 
 
-def find_triplets_with_0_sum(nums: list[int]) -> list[list[int]]:
+def find_triplets_with_0_sum(nums: list) -> list:
```

Comment:
Why?!?

Reply:
Since after updating the code in the existing file, "(nums: list[int]) -> list[list[int]]:" was failing in ruff test.
(ruff find_triplets_with_0_sum.py was throwing error so I changed to ( nums: list) -> list: )